main.py

```diff
@@ -63,25 +63,16 @@ def main():
             target_node_id = ox.nearest_nodes(graph, destination.x, destination.y)
             # wyznaczam i wyświetlam najkrótszą drogę od węzła wyjściowego do docelowego
             route = nx.shortest_path(G=graph, source=orig_node_id, target=target_node_id, weight="length")
-            ox.plot_graph_route(graph, route)
+            ox.plot_graph_route(graph, route)# pokazuje graf(?)
             # jeżeli wpłynęła jakaś prośba, to przypisuję ją do losowego auta; ale to jest rozwiązanie tymczasowe!
             # oczywiście trzeba to pozmieniać, bo w tym momencie auto może porzucić swoją dotychczasową prośbę
             # i przeteleportować się do punktu wyjściowego nowej prośby, co jest oczywiście nierealne
             random.choice(car_list).set_route(route)
         # update_position() służy do tego, żeby autko przesunęło się do przypisanej do niego drodze;
         # show_statistics() pokazuje, gdzie się obecnie znajduje
-        car_list[0].update_position()
-        car_list[0].show_statistics()
-        car_list[1].update_position()
-        car_list[1].show_statistics()
-        car_list[2].update_position()
-        car_list[2].show_statistics()
-        car_list[3].update_position()
-        car_list[3].show_statistics()
-        car_list[4].update_position()
-        car_list[4].show_statistics()
+        update_all(car_list)

```

Comment:
...to działa tak jak trzeba. Przez większość czasu auta są w miejscu, jest 1/50 szansy żeby każde auto dostało przydzieloną jakąś prośbę i to działa.